File: src/main/java/uk/gov/justice/digital/hmpps/whereabouts/utils/AbsentReasonFormatter.java
package uk.gov.justice.digital.hmpps.whereabouts.utils;

import static org.apache.commons.lang3.StringUtils.*;

public class AbsentReasonFormatter {

    public static String titlecase(final String reason) {

        return capitalize(lowerCase(join(splitByCharacterTypeCamelCase(reason), ' ')));
    }
}

File: src/main/java/uk/gov/justice/digital/hmpps/whereabouts/utils/AbsentReasonFormatter.kt
package uk.gov.justice.digital.hmpps.whereabouts.utils

import org.apache.commons.lang3.StringUtils.capitalize
import org.apache.commons.lang3.StringUtils.join
import org.apache.commons.lang3.StringUtils.lowerCase
import org.apache.commons.lang3.StringUtils.splitByCharacterTypeCamelCase

object AbsentReasonFormatter {
  fun titlecase(reason: String?) = capitalize(lowerCase(join(splitByCharacterTypeCamelCase(reason), ' ')))
}
